### PR TITLE
#5771 offers tab issue

### LINKED ISSUE #5771: Offers tab on kusama bug
https://user-images.githubusercontent.com/90852205/233826828-13010336-0c25-4b55-9f5f-3f0d6f79b1f4.mov

- it happens when you doubleclick on offers - it should not be clickable in case of kusama 
tested here: https://canary.kodadot.xyz/rmrk/gallery/17522429-9AFFD71D65E3DBBF29-CONNECTION-CONNECTION_5-0000000000000005

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Closes #5771
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [ ] My contribution builds **clean without any errors or warnings**
- [ ] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [ ] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### <samp>🤖 Generated by Copilot at 921e317</samp>

Renamed and refactored the `DisabledTab` component and its usage in the `GalleryItemDescription` and `GalleryItemTabsPanel` components. This change improves the code clarity, consistency, and readability of the component naming convention and style.

### <samp>🤖 Generated by Copilot at 921e317</samp>

> _A component called `DisablableTab`_
> _Was renamed to `DisabledTab`_
> _But then someone thought_
> _That `TabDisabled` was hot_
> _And changed it again in a snap_

### CODE REVIEW

**Comment by exezbcz** [comment]
no tooltip on hover

**Review by roiLeo — CHANGES_REQUESTED** [review]
- `DisablableTab` means that we have a Tab that can have disabled state
- `DisabledTab` would mean to me that this tab is disabled

**Comment by prachi00** [comment]
> * DisablableTab

oh, DisablableTab seemed like a weird word, I thought it was a typo, I'll revert this after my local env starts working
Is it working for you guys? @roiLeo @Jarsen136

**Comment by daiagi** [comment]
> I'll revert this after my local env starts working Is it working for you guys? @roiLeo @Jarsen136

are you on Windows by any chance?

**Comment by prachi00** [comment]
> > I'll revert this after my local env starts working Is it working for you guys? @roiLeo @Jarsen136
> 
> are you on Windows by any chance?

no, using mac

File: libs/ui/src/components/DisabledTab/DisabledTab.scss [hunk 1]
```diff
@@ -4,7 +4,6 @@
   &--disabled {
     // overide o-tabs-item
     cursor: auto !important;
-    pointer-events: all !important;
```
Comment (daiagi) [inline libs/ui/src/components/DisabledTab/DisabledTab.scss:7]: this was put here in order for tooltip to show

**Comment by daiagi** [comment]
> I'll revert this after my local env starts working Is it working for you guys? @roiLeo @Jarsen136

Yes. local env is working

File: libs/ui/src/components/DisabledTab/DisabledTab.scss (hunk 1 above)
Comment (prachi00) [reply libs/ui/src/components/DisabledTab/DisabledTab.scss:7]: yeah I figured, adding this makes disabled tab it clickable though

Comment (prachi00) [reply libs/ui/src/components/DisabledTab/DisabledTab.scss]: any way to have pointer-events: none and still show tooltip? @daiagi @roiLeo

Comment (daiagi) [reply libs/ui/src/components/DisabledTab/DisabledTab.scss]: from the top of my head i think you will need to make the tooltip trigger parent of the tab header

Comment (prachi00) [reply libs/ui/src/components/DisabledTab/DisabledTab.scss]: yeah @click.prevent didnt work, tried other ways as well, not able to make it work, maybe we can close this issue and someone else can take over this one

**Pull request closed without merging**